### PR TITLE
Feat/bac 70 build update channel endpoint

### PR DESCRIPTION
**Pull request by nodebe**
- Had a little bug in code, was return data.dict() so I had to remove the dict() function.
- All is working perfectly now.

### CODE REVIEW

File: backend/utils/message_utils.py [hunk 1]
```diff
@@ -6,7 +6,11 @@
 from utils.paginator import off_set
 
 
+<<<<<<< HEAD
+async def get_org_messages(org_id: str) -> Optional[List[Dict[str, Any]]]:
+=======
 async def get_org_messages(org_id: str, page: int, limit: int) -> Optional[List[Dict[str, Any]]]:
+>>>>>>> cb63f47124335054616951996835992142a5dc15
```
Comment (mikengr) [inline backend/utils/message_utils.py]: These can't be in your PR and you say all is working fine @nodebe

File: backend/utils/message_utils.py [hunk 2]
```diff
@@ -44,7 +48,11 @@ async def get_org_messages(org_id: str, page: int, limit: int) -> Optional[List[
 
 
 async def get_room_messages(
+<<<<<<< HEAD
+    org_id: str, room_id: str
+=======
     org_id: str, room_id: str, page: int, limit: int
+>>>>>>> cb63f47124335054616951996835992142a5dc15
 ) -> Optional[List[Dict[str, Any]]]:
```
Comment (mikengr) [inline backend/utils/message_utils.py]: same here @nodebe

**Review by mikengr — CHANGES_REQUESTED** [review]
@nodebe

File: backend/utils/message_utils.py (hunk 2 above)
Comment (nodebe) [reply backend/utils/message_utils.py]: done. @mikengr

**Review by mikengr — APPROVED** [review]
LGTM

**Pull request merged**